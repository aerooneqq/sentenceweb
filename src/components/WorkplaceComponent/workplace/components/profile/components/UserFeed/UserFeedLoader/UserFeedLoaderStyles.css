.userFeedLoaderContainer { 
    width: 100%;

    display: flex;
    flex-direction: column;

    justify-content: center; 
    align-items: center;
}

.userFeedLoaderElement {
    margin-top: 20px;

    width: 95%;

    display: flex; 
    flex-direction: row;
}

.userFeedLoaderUserPhoto { 
    width: 50px; 
    height: 50px;

    background: rgba(249, 118, 70, 0.5);

    border-radius: 50px;
}

.userFeedLoaderTextContainer { 
    flex-grow: 1;

    height: 100%;

    margin-left: 10px;
}

.userFeedLoaderTextDate { 
    width: 40%;
    height: 20px;

    background: #e0e0e0; 

    border-radius: 5px;
}

.userFeedLoaderTextContent { 
    margin-top: 10px;

    width: 60%;
    height: 30px;

    background: #d0d0d0;

    border-radius: 5px;
}