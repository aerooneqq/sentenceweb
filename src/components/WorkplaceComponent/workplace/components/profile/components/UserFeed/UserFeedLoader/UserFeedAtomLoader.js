import React, {Component} from "react";

import "./UserFeedLoaderStyles.css";

export default class UserFeedAtomLoader extends Component { 

    constructor(props) { 
        super(props);
    }

    render() { 
        return (
            <div className = "userFeedLoaderElement">
                <div className = "userFeedLoaderUserPhoto" />
                <div className = "userFeedLoaderTextContainer">
                    <div className = "userFeedLoaderTextDate" />
                    <div className = "userFeedLoaderTextContent" />
                </div>
            </div>
        )
    }

}