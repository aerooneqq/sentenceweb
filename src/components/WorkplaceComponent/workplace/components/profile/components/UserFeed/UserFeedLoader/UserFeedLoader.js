import React, {Component} from "react";

import "./UserFeedLoaderStyles.css";
import UserFeedAtomLoader from "./UserFeedAtomLoader";

export default class UserFeedLoader extends Component { 

    constructor(props) { 
        super(props);
    }

    render() { 
        return (
            <div className = "userFeedLoaderContainer">

                <UserFeedAtomLoader />
                <UserFeedAtomLoader />
                <UserFeedAtomLoader />
                <UserFeedAtomLoader />

            </div>
        )
    }

}