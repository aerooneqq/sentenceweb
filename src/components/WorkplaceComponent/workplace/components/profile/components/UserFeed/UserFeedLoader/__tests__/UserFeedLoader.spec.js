import React from "react";
import {shallow} from "enzyme";
import {shallowToJson} from "enzyme-to-json";
import UserFeedLoader from "../UserFeedLoader";
import UserFeedAtomLoader from "../UserFeedAtomLoader";

describe("UserFeedLoader", () => {
    it("should render correctly", () => {
        expect(shallowToJson(shallow(<UserFeedLoader />))).toMatchSnapshot();
    });

    it("should have 4 UserFeedAtomLoader", () => { 
        const userFeedLoader = shallow(<UserFeedLoader />);

        expect(userFeedLoader.find(UserFeedAtomLoader).length).toBe(4);
    });
});