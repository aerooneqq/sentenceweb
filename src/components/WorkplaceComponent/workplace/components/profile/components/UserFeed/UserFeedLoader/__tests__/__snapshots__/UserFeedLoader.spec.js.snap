// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserFeedLoader should render correctly 1`] = `
<div
  className="userFeedLoaderContainer"
>
  <UserFeedAtomLoader />
  <UserFeedAtomLoader />
  <UserFeedAtomLoader />
  <UserFeedAtomLoader />
</div>
`;
